databricks/koalas/tests/test_series.py

```diff
@@ -244,6 +246,13 @@ def test_nunique(self):
         self.assertEqual(ks.Series(range(100)).nunique(approx=True, rsd=0.01), 100)
 
     def test_value_counts(self):
+        set_arrow_conf = False
+        if LooseVersion(pyspark.__version__) < LooseVersion("2.4") and \
+                default_session().conf.get("spark.sql.execution.arrow.enabled") == "true":
+            default_session().conf.set("spark.sql.execution.arrow.enabled", "false")
+            set_arrow_conf = True
```

Comment:
Why do we need this?
If we need this, could you make sure to set back to the original conf?

Reply:
we need this to test at `pyspark2.3` same reason with https://github.com/databricks/koalas/pull/1116#discussion_r357451741,

i fixed the test logic to make sure setting back to original conf value.

Thanks for the review :)

Reply:
For now, i've fixed it to the current implementation.